README.md

```diff
@@ -60,6 +56,12 @@ Starting the server without any argument is the same as
 
 The stream can then be accessed at `http://<IP of the server>:8080/stream`
 
+### Maximum resolution
+
+Please note that the maximum recommended resolution is 1920x1080 (16:9).
+
+The absolute maximum resolution is 1920x1920. If you choose a higher resolution spyglass may crash.
+
 ## Using Spyglass with Mainsail
 
 If you ant to use Spyglass as a webcam source for [Mainsail]() add a webcam with the following configuration:
```

Comment:
If you are already fixing some other docs stuff this typo on line 67 can be fixed too

Reply:
`want` instead of `ant`